### DISCUSSION TITLE
Option to Hide moved lines as if lines are identical

### DISCUSSION THREAD

**Discussion by lecriquet38** [WinMerge/winmerge#1035] [Ideas]
Hello, 
When comparing 2 files, Winmerge spot the moved lines.

I have looked everywhere but could not find an option to hide those moved lines.

I do not care about those moved lines, i only care that they are in both file!
For instance: GRANT rights in .sql files. Those GRANT lines can be everywhere in both files, i do not matter.

Can you add an option to get rid of those lines like the identical ones when we press CTRL + D ?

Thanks

**Comment by BaronWilliams** [comment]
I also need this feature.

Thanks.
